toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -843,7 +843,7 @@ func hydrateFiles(fileTypeToHydrate fileType, specFile, workingDir string, srcCo
 		err = hydrateFromLocalSource(fileHydrationState, newSourceDir, srcConfig, skipSignatureHandling, currentSignatures)
 		// On error warn and default to hydrating from an external server.
```

Comment:
Comment isn't accurate anymore, we now error out.

Reply:
I think it was never entirely accurate. I've updated it and the `hydrateFromLocalSource` function + its comment as well.